demos/Shrine/Shrine/AppDelegate.swift

```diff
@@ -24,7 +24,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
   func application(_ application: UIApplication, didFinishLaunchingWithOptions
                    launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
-
+    window = UIWindow(frame: UIScreen.main.bounds)
     let flexHeadVC = ShrineFlexibleHeaderContainerViewController()
     self.window?.rootViewController = flexHeadVC
```

Comment:
Why not `self.window` above? Consistent with the lines below?

Reply:
swift should be selfless, removing other selfs :)